modules/assim.sequential/man/calculate_metrics.Rd

```diff
@@ -0,0 +1,25 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/downscale_function.R
+\name{calculate_metrics}
```

Comment:
`make document` should update this to the new name

Reply:
Addressed the issue in [71c1013](https://github.com/PecanProject/pecan/pull/3321/commits/71c1013e6816457afa3f95a4180f73f8c869b4f9) commit . Also took care of the existing calculate_metrics.Rd file in the previous commit [3889397](https://github.com/PecanProject/pecan/pull/3321/commits/388939742f559760d6fbfc1e8b51596858462fba)